textproc/gpsbabel/Portfile

```diff
@@ -26,63 +21,57 @@ long_description    GPSBabel converts GPS waypoints, tracks, and routes \
 platforms           darwin
 homepage            https://www.gpsbabel.org
 
-depends_lib-append  port:expat \
-                    port:libusb-compat
+checksums           rmd160  dd41329723a41749a6ef929e0766fba708509116 \
+                    sha256  a5ef3b1744488d86d438291df2d65135cb04c435680fe7f64f5a94a3814ebea4 \
+                    size 16720596
 
-checksums           rmd160  b38b81b38cd50cc4d1c36faa962afb02edb72cd3 \
-                    sha256  fbec8cc36ff4b361e3eada286d3018d34579f919ac5398988e5e85061153fa22
-
-compiler.cxx_standard   2011
+compiler.cxx_standard   2014
 
 if {${subport} eq ${name}} {
 
-    patchfiles-append  patch-delbin.cc.diff
-
-    # use MacPorts libraries
-    # see https://gitweb.gentoo.org/repo/gentoo.git/tree/sci-geosciences/gpsbabel
-    patchfiles-append  patch-macports_libs.diff
-    use_autoreconf      yes
-    # see https://github.com/gpsbabel/gpsbabel/commit/15f8b8b6ec5145a56d06d509b2e90076f75dd867
-    # see https://github.com/gpsbabel/gpsbabel/commit/30a345577a644125502f6e4f8b58ea3077ec8dad
-    patchfiles-append  patch-qt5.diff
-    # avoid error: ordered comparison between pointer and zero ('const char *' and 'int')
-    patchfiles-append  patch-bushnell.cc.diff
-
-    configure.universal_args-delete \
-        --disable-dependency-tracking
+    # configure with qmake
+    # see https://github.com/GPSBabel/gpsbabel/pull/611
+    patchfiles-append patch-pr611.diff
+    patch.pre_args -p1
 
     depends_lib-append  \
+        port:libusb     \
```

Comment:
Other ports allow other versions of libusb to be used. Can you test and use this instead?

`path:lib/pkgconfig/libusb-1.0.pc:libusb
`

```
$ port list libusb*
libusb                         @1.0.24         devel/libusb
libusb-compat                  @0.1.7          devel/libusb-compat
libusb-devel                   @20210217-d97e85ef devel/libusb
libusb-legacy                  @0.1.12         devel/libusb-legacy
```

Reply:
Your suggestion allows libusb (1.0.24) or libusb-devel (20210217-d97e85ef) to be used.  I'll push the change.

Reply:
Note I also added "qt5.min_version         5.9", which is the minimum version of Qt that gpsbabel 1.7.0 requires.